playground/AzureContainerApps/AzureContainerApps.AppHost/infra.module.bicep

```diff
@@ -75,7 +75,6 @@ resource cae_Contributor 'Microsoft.Authorization/roleAssignments@2022-04-01' =
   properties: {
     principalId: userPrincipalId
     roleDefinitionId: subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
-    principalType: 'ServicePrincipal'
```

Comment:
@davidfowl - was this caused by one of your changes? Was it intentional? I don't think my change could have affected this.

Reply:
yea it was my change